features/crashprobe.feature

```diff
@@ -7,40 +7,55 @@ Feature: Reporting crash events
     When I run "PrivilegedInstructionScenario" and relaunch the app
     And I configure Bugsnag for "PrivilegedInstructionScenario"
     And I wait to receive an error
-    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    Then the error is valid for the error reporting API
     And the error payload field "events" is an array with 1 elements
-    And the exception "errorClass" equals "EXC_BAD_INSTRUCTION"
+    And the exception "errorClass" equals one of:
+      | Intel | EXC_BAD_ACCESS      |
+      | ARM   | EXC_BAD_INSTRUCTION |
     And the "method" of stack frame 0 equals "-[PrivilegedInstructionScenario run]"
 
   Scenario: Calling __builtin_trap()
     When I run "BuiltinTrapScenario" and relaunch the app
     And I configure Bugsnag for "BuiltinTrapScenario"
     And I wait to receive an error
-    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    Then the error is valid for the error reporting API
     And the error payload field "events" is an array with 1 elements
-    And the exception "errorClass" equals "EXC_BREAKPOINT"
+    And the exception "errorClass" equals one of:
+      | Intel | EXC_BAD_INSTRUCTION |
+      | ARM   | EXC_BREAKPOINT      |
     And the "method" of stack frame 0 equals "-[BuiltinTrapScenario run]"
 
   Scenario: Calling non-existent method
     When I run "NonExistentMethodScenario" and relaunch the app
     And I configure Bugsnag for "NonExistentMethodScenario"
     And I wait to receive an error
-    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    Then the error is valid for the error reporting API
     And the error payload field "events" is an array with 1 elements
-    And the exception "message" starts with "-[NonExistentMethodScenario santaclaus:]: unrecognized selector sent to instance"
+    # TODO: Figure out why message is empty on macOS
+    #And the exception "message" equals the platform-dependent string:
+    #  | ios   | -[NonExistentMethodScenario santaclaus:]: unrecognized selector sent to instance |
+    #  | macos | @skip                                                                            |
```

Comment:
What was the outcome here?  I'm happy to raise tickets for follow-on work, so long as we reference the ticket numbers.

Reply:
@nickdowell do you have any info on this?  Would you be able to create a ticket and I'll reference it here?

Reply:
Afraid I don't remember the details of this one; have raised ticket PLAT-5860 since it seems worth further investigation.